packages/twenty-front/src/utils/recoil/cssVariableEffect.ts

```diff
@@ -0,0 +1,22 @@
+import { type AtomEffect } from 'recoil';
+
+export const cssVariableEffect =
```

Comment:
Let's use an EffectComponent like `RecordTableColumnWidthEffect` to avoid creating another pattern for the same type of logic. The less patterns we have in the code base the better.